Userland/Libraries/LibWeb/DOM/StyleElementUtils.cpp

```diff
@@ -7,9 +7,11 @@
 #include <LibWeb/CSS/Parser/Parser.h>
 #include <LibWeb/CSS/StyleComputer.h>
 #include <LibWeb/DOM/Document.h>
+#include <LibWeb/DOM/Element.h>
 #include <LibWeb/DOM/ShadowRoot.h>
 #include <LibWeb/DOM/StyleElementUtils.h>
 #include <LibWeb/Infra/Strings.h>
+#include <cassert>
```

Comment:
Use VERIFY() for runtime asserts and ASSERT() for debug asserts (runtime asserts preferred). We don't include STL headers.